model/riscv_csr_map.sail

```diff
@@ -154,11 +164,20 @@ mapping clause csr_name_map = 0x3BC  <-> "pmpaddr12"
 mapping clause csr_name_map = 0x3BD  <-> "pmpaddr13"
 mapping clause csr_name_map = 0x3BE  <-> "pmpaddr14"
 mapping clause csr_name_map = 0x3BF  <-> "pmpaddr15"
+mapping clause csr_name_map = 0x721  <-> "mcyclecfgh"
+mapping clause csr_name_map = 0x722  <-> "minstretcfgh"
 /* machine counters/timers */
 mapping clause csr_name_map = 0xB00  <-> "mcycle"
 mapping clause csr_name_map = 0xB02  <-> "minstret"
 mapping clause csr_name_map = 0xB80  <-> "mcycleh"
 mapping clause csr_name_map = 0xB82  <-> "minstreth"
+mapping clause csr_name_map = 0x350  <-> "miselect"
+mapping clause csr_name_map = 0x351  <-> "mireg"
+mapping clause csr_name_map = 0x352  <-> "mireg2"
+mapping clause csr_name_map = 0x353  <-> "mireg3"
+mapping clause csr_name_map = 0x355  <-> "mireg4"
+mapping clause csr_name_map = 0x356  <-> "mireg5"
+mapping clause csr_name_map = 0x357  <-> "mireg6"
 /* TODO: other hpm counters and events */
```

Comment:
To keep the list sorted by address, the indirect CSRs are better placed between `mip` and `pmpcfg0`.